www/includes/easyparliament/templates/html/mp/divisions.php

```diff
@@ -0,0 +1,206 @@
+                <div class="person-search">
+                    <form action="<?= $search_url ?>" method="get" onsubmit="trackFormSubmit(this, 'Search', 'Submit', 'Person'); return false;">
+                        <input id="person_search_input" name="q" maxlength="200" placeholder="Search this person's speeches"><input type="submit" class="submit" value="GO">
+                        <input type="hidden" name="pid" value="<?= $person_id ?>">
+                    </form>
+                </div>
+                <div class="person-buttons">
+                  <?php if ($current_member_anywhere) { ?>
+                    <a href="https://www.writetothem.com/<?php
+                        if ($current_member[HOUSE_TYPE_LORDS]) {
+                            echo "?person=uk.org.publicwhip/person/$person_id";
+                        }
+                        if ($the_users_mp) {
+                            echo "?a=WMC&amp;pc=" . _htmlentities(urlencode($user_postcode));
+                        }
+                    ?>" class="button wtt" onclick="trackLinkClick(this, 'Links', 'WriteToThem', 'Person'); return false;"><img src="/style/img/envelope.png">Send a message</a>
+
+                  <?php } ?>
+                  <?php if ($has_email_alerts) { ?>
+                    <a href="<?= WEBPATH ?>alert/?pid=<?= $person_id ?>#" class="button alert" onclick="trackLinkClick(this, 'Alert', 'Search', 'Person'); return false;"><img src="/style/img/plus-circle.png">Get email updates</a>
+                  <?php } ?>
+                </div>
+            </div>
+            </div>
+        </div>
+    </div>
+
+<div class="full-page">
+    <div class="full-page__row">
+        <div class="full-page__unit">
+            <div class="person-navigation page-content__row">
+                <ul>
+                    <li><a href="<?= $member_url ?>">Overview</a></li>
+                    <li class="active"><a href="<?= $member_url ?>/votes">Voting Record</a></li>
+                </ul>
+            </div>
+            <div class="person-panels page-content__row">
+                <div class="sidebar__unit in-page-nav">
+                    <?php if ( isset($policyinformation) && isset($policies) && $policies && isset($policydivisions) && $policydivisions ) { ?>
+                    <p class="policy-votes-intro">
+                        How <?= $full_name ?> voted on <?= $policies[$policydivisions[array_keys($policydivisions)[0]]['policy_id']] ?>.
+                    </p>
+                    <?php } ?>
+                    <ul>
+                        <li><a href="<?= $member_url ?>/votes">Back to all topics</a></li>
+                    </ul>
+                </div>
+
+                <div class="primary-content__unit">
+
+                    <?php if ($party == 'Sinn Fein' && in_array(HOUSE_TYPE_COMMONS, $houses)): ?>
+                    <div class="panel">
+                        <p>Sinn F&eacute;in MPs do not take their seats in Parliament.</p>
```

Comment:
Do we want the SNP line about only voting on Scotland things here too? (Will that apply after the election, fact fans?) In fact, why is the SNP line treated differently to this line in the code, hmm.